README.md

```diff
@@ -215,13 +189,13 @@ docker run -d \
   -p 9998:9998 \
   $MF_DOCKER_USER/mock-firebolt
 
-# Run the image, enabling the manage and discovery SDKs (in addition to the core SDK) (not typical)
+# Run the image, enabling the discovery SDK (not typical)
```

Comment:
Same question as above... if/when discovery is a thing, it'll be in the one and only one OpenRPC file, right?